mkdocs_monorepo_plugin/plugin.py

```diff
@@ -15,6 +15,7 @@
 from mkdocs.plugins import BasePlugin
 from .parser import Parser
 from .merger import Merger
+from .helpers import is_index, set_edit_url
```

Comment:
Since all of these are related to fixing site edit URLs, does it make sense to name something more specific like `edit_url.py` instead of `helpers.py`?